README.md

```diff
@@ -249,17 +234,17 @@ The following environment variables must be set when deploying the app.
 | SUBMISSION_BUCKET_NAME                    | The name of the bucket that submissions will be stored in                            |
 | DOCKER_REGISTRY                           | The FQDN of the target Docker registry                                               |
 | IMAGE_TAG                                 |                                                                                      |
-| REQUESTED_CPU_PER_POD                     | No. of CPUs to request per Pod                                                       |
-| MIN_REPLICAS                              | Minimum no. of replicated Pods                                                       |
-| MAX_REPLICAS                              | Maximum no. of replicated Pods                                                       |
 
 The following environment variables are optional:
 
 | Variable Name                      | Default | Description                                                                       |
 |------------------------------------| --------|-----------------------------------------------------------------------------------|
-| ROLLING_UPDATE_MAX_UNAVAILABLE     | 25%     | The maximum number of Pods that can be unavailable during the update process     |
-| ROLLING_UPDATE_MAX_SURGE           | 25%     | The maximum number of Pods that can be created over the desired number of Pods   |
-| TARGET_CPU_UTILIZATION_PERCENTAGE  |         | The average CPU utilization usage before auto scaling applies                    |
+| REGION                             |         |                                                                                   |
+| CONCURRENCY                        |         |                                                                                   |
+| MIN_INSTANCES                      |         |                                                                                   |
+| MAX_INSTANCES                      |         |                                                                                   |
+| CPU                                |         |                                                                                   |
+| MEMORY                             |         |                                                                                   |
```

Comment:
The defaults and descriptions are missing.

Reply:
Also the existing variables need to be checked for correctness e.g. the default for `WEB_SERVER_TYPE` is `gunicorn-async` rather than `gunicorn-threads`.

Reply:
Added defaults, descriptions and revisited old values to check for changes, thanks!